crates/bevy_ecs/src/query/filter.rs

```diff
@@ -616,18 +616,27 @@ impl_tick_filter!(
 );
 
 impl_tick_filter!(
-    /// Filter that retrieves components of type `T` that have been changed since the last
-    /// execution of this system.
+    /// Filter that retrieves components of type `T` that have been changed since the last tick.
     ///
-    /// This filter is useful for synchronizing components, and as a performance optimization as it
-    /// means that the query contains fewer items for a system to iterate over.
+    /// All components internally remember the last tick they were added at, and the last tick they
+    /// were mutated at. The mutation detection is powered by `DerefMut`, such that any dereferencing
+    /// will mark the component as changed. Note that there is no deep value inspection on the actual
+    /// data of the component, which would be prohibilively expensive. This means false positives can
```

Comment:
```suggestion
    /// data of the component, which would be prohibitively expensive. This means false positives can
```

Reply:
Please tell me that's a bot, or you're the fastest reader ever 😂